pkgs/top-level/python-packages.nix

```diff
@@ -24794,10 +24794,10 @@ EOF
   };
 
   ofxhome = buildPythonPackage rec {
```

Comment:
can you move this expression. See the header of this file.

Reply:
Done.  Please take a look at new commit.